packs/data/feats.db/devils-advocate.json

```
@@ -11,19 +11,42 @@
             "value": null
         },
         "description": {
-            "value": "<p><strong>Access</strong> Chelaxian nationality</p>\n<hr />\n<p>You know more about the habits of devils than is entirely safe. You gain a +2 circumstance bonus to Perception checks against devils and saving throws against their abilities. In addition, whenever you meet a devil in a social situation, you can immediately attempt a Diplomacy check to Make an Impression on that creature rather than needing to converse for 1 minute. You take a -5 penalty to the check. If you fail, you can engage in 1 minute of conversation and attempt a new check at the end of that time rather than accepting the failure or critical failure result.</p>\n<p><strong>Special</strong> If you have the @Compendium[pf2e.feats-srd.Glad-Hand]{Glad-Hand} skill feat, you don't take the penalty to your immediate Diplomacy check if the target is a devil.</p>"
+            "value": "<p><strong>Access</strong> Chelaxian nationality</p>\n<hr />\n<p>You know more about the habits of devils than is entirely safe. You gain a +2 circumstance bonus to Perception checks against devils and saving throws against their abilities.</p>\n<p>In addition, whenever you meet a devil in a social situation, you can immediately attempt a Diplomacy check to Make an Impression on that creature rather than needing to converse for 1 minute. You take a -5 penalty to the check. If you fail, you can engage in 1 minute of conversation and attempt a new check at the end of that time rather than accepting the failure or critical failure result.</p>\n<p><strong>Special</strong> If you have the @Compendium[pf2e.feats-srd.Glad-Hand]{Glad-Hand} skill feat, you don't take the penalty to your immediate Diplomacy check if the target is a devil.</p>"
         },
         "featType": {
             "value": "ancestry"
         },
         "level": {
             "value": 1
         },
-        "location": "",
+        "location": null,
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": {
+                    "all": [
+                        "target:trait:devil"
+                    ]
```

```suggestion
                    "any": [
                        "target:trait:devil",
                        "origin:trait:devil"
                    ]
```
This will work if a devil feints for example.

---

Hmm. This suggestion looks like it's going to break the json format though. So maybe manually do this.